policy/policy.go

```diff
@@ -22,11 +22,19 @@ func Detect(url string, dst string) (string, error) {
 }
 
 // ReadFiles returns all of the policy files (not including tests)
-// at the given path including its subdirectories.
-func ReadFiles(path string) ([]string, error) {
-	files, err := getPolicyFiles(path)
-	if err != nil {
-		return nil, fmt.Errorf("search rego files: %w", err)
+// at the given path(s) including its subdirectories.
+func ReadFiles(paths ...string) ([]string, error) {
```

Comment:
I just realised how similar this method and the ReadTestFiles method are at the moment. Maybe we can improve on this and remove the duplication between the two methods?

Reply:
Definitely! Only difference, apart from the filename filter, is whether to treat empty files as an error. Is there a reason `ReadFilesWithTests` shouldn't consider that an error?

Reply:
No I don't think so, I think that should similarly lead to an error. I can't imagine a reason why it shouldn't error.

Reply:
Pushed a rewrite where only the filter differs